scripts/UpdateMetaModule.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param
+(
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Path
+)
+
+Write-Output "Running $($MyInvocation.MyCommand.Name)"
+
+$nuspecPath = (Get-Childitem �Path $Path -Include *.nuspec -Recurse).FullName
+if(!$nuspecPath)
+{
+    throw "Unable to find the nuspec under path: $Path"
+}
+Write-Output "The path to nuspec: $nuspecPath"
+
+$psdFilePath = (Get-Childitem �Path $Path -Include *.psd1 -Recurse).FullName
+if(!$psdFilePath)
+{
+    throw "Unable to find the module file under path: $Path"
+}
+Write-Output "The path to the module file: $psdFilePath"
+
+Write-Output "Getting content for nuspec from $nuspecPath"
+$nuspecContent = [xml](Get-Content -Path $nuspecPath -Raw)
+if(!$nuspecContent)
+{
+    throw "Nuspec file is empty under path: $nuspecPath"
+}
+Write-Output "Found content for nuspec file"
+
+Write-Output "Get all dependencies under .NET Standard 2.0 Group"
+$dep = $nuspecContent.package.metadata.dependencies.group.dependency
+if(!$dep)
+{
+    throw "No dependencies found in the nuspec file"
+}
+Write-Output "Found dependencies $dep"
+
+Write-Output "Removing the attributes which are not required and appending the dependencies"
+$dep | % { $_.RemoveAttribute("exclude"); $nuspecContent.package.metadata.dependencies.AppendChild($_) }
```

Comment:
Don't you need the xmlns path too?

Reply:
Removing both attributes together doesn't work. I added the following to remove xmlns path separately:
$nuspec.package.RemoveAttribute("xmlns")